OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>..\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```

Comment:
All our csproj files (except the four mod dlls, strangely) output to `..\`, so I would be much happier if we kept this as it was.